scripts/velocity.py

```diff
@@ -1,5 +1,25 @@
 """
 Generates reports on recently-closed issues on ZenHub and GitHub.
+
+To run this script locally, printing the report to stdout, the
+following must be configured in your environment.local.py:
+* azul_velocity_github_token
+* azul_velocity_zenhub_key
+* azul_velocity_zenhub_pkey
+* azul_velocity_zenhub_url
+Instructions for acquiring these keys (including required scopes,
+etc.) are listed in environment.py.
+
+To run this script with GitHub Actions, the aforementioned variables
+must be defined as secrets. For more info on setting secrets, see
+https://docs.github.com/en/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets .
+
+To publish the generated report to a Gitlab wiki, using GitHub
```

Comment:
if the `gitlab_api_key` was not set and the script ran in a github action, what would happen? Would it just print the output to the log of the action that was ran?

Reply:
Not necessarily; with the provided action, the script would be written to disk and the wiki upload step would fail without printing the report.